### ISSUE TITLE
Can't speak again after stop voice on Web

### ISSUE THREAD

**Issue by kittyliujc** [dlutton/flutter_tts#402]
## 🐛 Bug Report

Can't speak again after stop voice on Web

### Expected behavior
speak out again

### Reproduction steps
FlutterTts flutterTts = FlutterTts();
flutterTts.speak('test voice');
flutterTts.stop();  // stop trigger onError, and status keep playing, and can't play again
flutterTts.speak('test voice again');   // <----------- no speak out

### Configuration

**Version:** 3.7.0

**Platform:** 
Web

**Comment by SteveBraich** [comment]
I am having the same issue.

I tried reinitializing the FlutterTts object before speaking again, but it still doesn't work.

```
flutterTts = FlutterTts(); 
flutterTts.speak(targetTextController.text);
```

**Comment by khjde1207** [comment]
https://github.com/khjde1207/flutter_tts/blob/14b3676b0a49531d2b270a0f669cda0c0ca739da/lib/flutter_tts_web.dart#L96

It can be solved by correcting the error part as above.

**Comment by dlutton** [comment]
Thanks for the assistance @khjde1207 , this is fixed in 3.8.2.

**Comment by digiperfect-janak** [comment]
> Thanks for the assistance @khjde1207 , this is fixed in 3.8.2.

Hi,

Seems to be happening even in 4.2.1. Speak does not work for some time after stop.  There is a log emmitted. [object SpeechSynthesisErrorEvent]
After few seconds, speak works again.
